docs/modules/ROOT/pages/explanation/decisions/dns-nameing.adoc

```diff
@@ -0,0 +1,39 @@
+= DNS nameing scheme
+
+Problem::
+{product} is a web based service offering.
+As such it is accessible through several domains.
+We need a common understanding how we name things and what domains to use for making it accessible.
+
+Alternatives::
+* In the past, we used different domains for things managed by us and things deployed by users. This due to a concern regarding cross domain cookie access. 
```

Comment:
The last sentence sounds strange I would rewrite it like this:
`... things deployed by users, due to a concern ...